schema/geometry_schema.json

```diff
@@ -94,6 +94,10 @@
 						},
 						"north_axis": {
 							"type": "number"
+						},
```

Comment:
I would like to move north_axis and building_min_z out of project->config, should be in something like a building object.  @bgschiller are you using these two fields yet?

Reply:
We're using north_axis for when users rotate the map. Is the editor supposed to support multiple buildings? It seems like `north_axis` needs to be the same among all entities in a project.

Reply:
That's correct, so maybe we just create a building object for the min z